manifests/pulp_client.pp

```diff
@@ -0,0 +1,48 @@
+# Pulp Client Certs
+class certs::pulp_client (
+  $hostname                          = $::certs::node_fqdn,
+  $generate                          = $::certs::generate,
+  $regenerate                        = $::certs::regenerate,
+  $deploy                            = $::certs::deploy,
+
+  $common_name                       = $certs::common_name
```

Comment:
Nitpick: make it consistent with the others and use ```$::certs```. I'm also a fan of trailing comma's because it keeps the diff smaller if another parameter is added.